include/tvm/runtime/packed_func.h

```diff
@@ -674,10 +675,12 @@ inline TVMType String2TVMType(std::string s) {
     scan = s.c_str();
     LOG(FATAL) << "unknown type " << s;
   }
-  unsigned bits = t.bits, lanes = t.lanes;
-  sscanf(scan, "%ux%u", &bits, &lanes);
-  t.bits = static_cast<uint8_t>(bits);
-  t.lanes = static_cast<uint16_t>(lanes);
+  unsigned bits, lanes;
```

Comment:
This code need to be change to support all possible cases

- ```int```  (no bits and lanes)
- ```int32``` (only bits, no lanes)
- ```int32x4``` (both bits and lanes)

so far it only support the last version

Reply:
This can be the main reason causing the CI to fail